Makefile

```diff
@@ -40,7 +40,10 @@ po/$(PACKAGE_NAME).html.pot: $(NODE_MODULES_TEST)
 po/$(PACKAGE_NAME).manifest.pot: $(NODE_MODULES_TEST)
 	po/manifest2po src/manifest.json -o $@
 
-po/$(PACKAGE_NAME).pot: po/$(PACKAGE_NAME).html.pot po/$(PACKAGE_NAME).js.pot po/$(PACKAGE_NAME).manifest.pot
+po/$(PACKAGE_NAME).metainfo.pot: org.cockpit-project.$(PACKAGE_NAME).metainfo.xml
+	xgettext --default-domain=cockpit --output=$@ $<
```

Comment:
Urgh, we shouldn't call the domain "cockpit", but `$(PACKAGE_NAME)`, like the .pot file. We already do that in the .js.pot rule, so let's fix this in a separate commit/PR. I'm fine to do that, unless you want to add it in this PR. But let's use the correct name right away here.

Reply:
I can do a separate PR for it, no problem; note that in the end the domain name does not matter too much, as translations are either merged back to other files (eg the AppStream metadata) or converted to other formats (JSON).

Reply:
it is #536